met-web/src/components/engagement/form/EngagementWidgets/Documents/FileDrawer.tsx

```diff
@@ -140,10 +174,13 @@ const FileDrawer = () => {
                                     InputLabelProps={{
                                         shrink: false,
                                     }}
+                                    defaultValue={parentDocument?.title}
                                     fullWidth
                                     size="small"
                                 >
-                                    <MenuItem key={`folder-option-0`} value={0} sx={{ height: '2em' }}></MenuItem>
+                                    <MenuItem key={`folder-option-0`} value={0} sx={{ height: '2em' }}>
+                                        none
```

Comment:
kinda liked the empty space, if a 'none' text here is preferred maybe italicize it?

Reply:
italicized